src/main/resources/public/index.html

```diff
@@ -12,10 +12,19 @@
         <button>login</button>
         <button>sign up</button>
     </div>
-    <h1>Welcome to Moloko Blog!!!</h1>
-    <form id="create-article-form">
+    <form action="article/export">
+        <button>Export</button>
+    </form>
+
+    <form id="import-articles-form">
+        <button type="button" id="importBtn">Import</button>
```

Comment:
can we be more consistent and use either Btn or button word

Reply:
also we should agree on style for html IDs, classes. There is mismatch between this id (camelCase) and id for file input (id with dash as word separator)